view/layout.dust

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Can we add tracking tags from the likes of Google Tag Manager?

See the existing Pugin layout for requirements